templates/FFV_form_factors.cpp.in

```diff
@@ -429,14 +479,91 @@ double OneLoopFunctionN(double r, double s)
       } else if (is_zero(1.0 - r)){
          return 2.0 / 3.0;
       } else {
-         return (1.0 - 4.0 * r + 3 * r * r - 2 * r * r * std::log(r)) / Power3(1.0 - r);
+         return (1.0 - 4.0 * r + 3 * r * r - 2 * r * r * std::log(r)) / 2.0 / Power3(1.0 - r);
       }
+   } else if (1./s < ratiolimit) {
+      // Expand s about 0 to 1st order
+      return (1.0 - r * r * s + r * (s - 1.0) + r * (s + 1.0) * std::log(r)) / Power2(r - 1.0);
    } else {
       return s * (r * Power2(r - s) * std::log(r) + (r - 1.0) * ((r - s) * (s - 1.0) - (r - 1.0)
                               * r * std::log(r/s))) / Power2(1.0 - r) / Power2(r - s) / (s - 1.0);
    }
 }
 
+
+double Kallenlambda(double m1, double m2, double m3)
```

Comment:
Would it be possible to use `KallenLambda` from `src/wrappers.hpp`?

Reply:
I do not see a <KallenLambda> function in <src/wrappers.hpp>, or do you mean move it to there?

Reply:
Ah, sorry, it is in the `FlexibleDecay-merge_upstream2` branch, which has not yet been merged into `development`.

Ok, then I'd propose to leave your function here. But could you please rename it to `KallenLambda`?
You may also want to simplify it like so:
~~~.cpp
template<typename T>
T KallenLambda(T x, T y, T z)
{
   return Sqr(x-y-z) - 4*y*z;
}
~~~

Reply:
Okay, this is done.  Thank you.